tests/bins_test.py

```diff
@@ -386,3 +386,12 @@ def test_bins_like():
     with pytest.raises(sc.DimensionError):
         dense = dense.rename_dims({'x': 'y'})
         sc.bins_like(binned, dense),
+
+
+def test_bins_concat():
+    table = sc.data.table_xyz(nrow=100)
+    table.data = sc.arange('row', 100, dtype='float64')
+    da = table.bin(x=4, y=5)
+    assert sc.identical(da.bins.concat('x').hist(), table.hist(y=5))
+    assert sc.identical(da.bins.concat('y').hist(), table.hist(x=4))
+    assert sc.identical(da.bins.concat().hist(), table.sum())
```

Comment:
Also `assert da.bins.concat().bins is None`?

Reply:
Hmm, but it is not. It preserves a single bin (0-D variable).

Reply:
Should it do that?

Reply:
I don't think so. This would, e.g., drop meta data (such as bin edges), or attempt to merge it into the inner meta data. Furthermore it seems inconsistent?

We *do* need to consider providing a function that does that though.